src/scripts/main.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const div = document.createElement('div');
+
+  div.classList.add('notification', type);
+  div.style.top = `${posTop}px`;
+  div.style.right = `${posRight}px`;
+
+  const h2 = document.createElement('h2');
+
+  h2.setAttribute('class', 'title');
+  h2.textContent = title;
+
+  const p = document.createElement('p');
+
+  p.textContent = description;
+
+  div.append(h2, p);
+  document.body.appendChild(div);
+
+  // Удаляем элемент через 2 секунды
+  setTimeout(() => {
+    div.style.visibility = 'hidden';
```

Comment:
Instead of setting `div.style.visibility = 'hidden';`, consider using `div.remove();` to completely remove the notification from the DOM. This will prevent unnecessary elements from accumulating in the DOM.